src/BasePostMessageStream.ts

```diff
@@ -28,6 +41,20 @@ export abstract class BasePostMessageStream extends Duplex {
     // Initialization flags
     this._init = false;
     this._haveSyn = false;
+    this._logger = {
+      source: '',
+      destination: '',
+      logger: () => null,
+    };
+  }
+
+  private _log(data: StreamData, out: boolean) {
+    this._logger.logger(
```

Comment:
Similar comment as https://github.com/MetaMask/extension-port-stream/pull/46#discussion_r1210744465.

Reply:
that's fair 
34da10e